.github/workflows/arm-runner.yml

```diff
@@ -57,11 +57,11 @@ jobs:
             sudo rm -rf /pynab
             sudo apt-get autoremove -y
             sudo apt-get clean
-    - name: Compress the release image with pishrink
+    - name: Compress the release image with PiShrink
       run: |
         mv ${{ steps.arm_runner_install.outputs.image }} pynab-${{ steps.image_and_branch_name.outputs.IMAGE_NAME_SUFFIX }}.img
-        wget https://raw.githubusercontent.com/Drewsif/PiShrink/fa3c7a27406e7c0a1c7b8688e330f14a766be1ba/pishrink.sh
-        sudo bash pishrink.sh -Z -a pynab-${{ steps.image_and_branch_name.outputs.IMAGE_NAME_SUFFIX }}.img
+        wget https://raw.githubusercontent.com/Drewsif/PiShrink/master/pishrink.sh
```

Comment:
We probably should not checkout master version of pishrink as it is not stable, could fail in the future and introduces a security dependecy.

Reply:
I moved back from master to the commit you previously select.
The real security isssue here is that PiShrink doesn't have a reliable release mechanism: maybe we should try and find an alternative?